spec/amqproxy_spec.cr

```diff
@@ -1,6 +1,75 @@
 require "./spec_helper"
 
 describe AMQProxy::Server do
+  it "graceful shutdown" do
+    started = Time.utc.to_unix
+
+    s = AMQProxy::Server.new("127.0.0.1", 5672, false, Logger::DEBUG, 5, true)
+    channel = Channel(Nil).new
```

Comment:
Maybe have a better name than the generic "channel"? That tries to describe the purpose of it

Reply:
Fixed by calling it `wait_for_channel`